### PR TITLE
additional info on heading() function

### PR DESCRIPTION
**Pull request by avenirer**
...worth mentioning...

### CODE REVIEW

File: user_guide_src/source/helpers/html_helper.rst [hunk 1]
```diff
@@ -27,11 +27,11 @@ The following functions are available:
 
 .. php:function:: heading([$data = ''[, $h = '1'[, $attributes = '']]])
 
-	:param	string	$data: Content
-	:param	string	$h: Heading level
-	:param	array	$attributes: HTML attributes
-	:returns:	HTML heading tag
-	:rtype:	string
+	:param	string		$data: Content
```
Comment (narfbg) [inline user_guide_src/source/helpers/html_helper.rst]: Don't realign these ...

File: user_guide_src/source/helpers/html_helper.rst [hunk 2]
```diff
@@ -27,11 +27,11 @@ The following functions are available:
...
+	:param	string		$h: Heading level
+	:param	array|string	$attributes: HTML attributes
```
Comment (narfbg) [inline user_guide_src/source/helpers/html_helper.rst]: Just 'mixed'

File: user_guide_src/source/helpers/html_helper.rst [hunk 3]
```diff
@@ -41,15 +41,18 @@ The following functions are available:
 	The above would produce: <h3>Welcome!</h3>
 
 	Additionally, in order to add attributes to the heading tag such as HTML
-	classes, ids or inline styles, a third parameter is available::
+	classes, ids or inline styles, a third parameter is available either
```
Comment (narfbg) [inline user_guide_src/source/helpers/html_helper.rst]: 'is available' should be changed to 'accepts' (and adjust accordingly the wording after that)

File: user_guide_src/source/helpers/html_helper.rst [hunk 4]
```diff
@@ -41,15 +41,18 @@ The following functions are available:
 	The above would produce: <h3>Welcome!</h3>
 
 	Additionally, in order to add attributes to the heading tag such as HTML
-	classes, ids or inline styles, a third parameter is available::
+	classes, ids or inline styles, a third parameter is available either 
+	as a string or as an array::
 
-		echo heading('Welcome!', 3, 'class="pink"')
+		echo heading('Welcome!', 3, 'class="pink"');
+		echo heading('How are you?', 4, array('id'=>'question', 'class'=>'green');
```
Comment (narfbg) [inline user_guide_src/source/helpers/html_helper.rst]: Add spaces on each side of `=>`

**Comment by avenirer** [comment]
Made the according changes. Thank you.

**Pull request merged**